storage/client/blob/src/samples/java/blob/StorageErrorHandlingExample.java

```diff
@@ -18,42 +19,42 @@
 public class StorageErrorHandlingExample {
 
     public static void main(String[] args) {
-        ContainerClient containerClient = ContainerClient.containerClientBuilder().endpoint("https://account.blob.core.windows.net/mycontainer")
+        ContainerClient containerClient = new ContainerClientBuilder().endpoint("https://account.blob.core.windows.net/mycontainer")
```

Comment:
Any reason to change like this way?

Reply:
The builder static on the client was removed based on #3871